### DISCUSSION TITLE
Add DFS index to list-windows output for easier scripting

### DISCUSSION THREAD

**Discussion by renke** [nikitabobko/AeroSpace#995] [feature-ideas]
Focus already supports targeting a window by the DFS index. To allow easier scripting it would be helpful to expose a window's current DFS index when using `list-windows`. Given that, one could easily implement something like https://github.com/nikitabobko/AeroSpace/issues/248 in userland.

**Comment by kenbanks-peng** [comment]
This would be nice

**Reply by mattcoleanderson** [reply]
Agreed
